src/backtrace.c

```diff
@@ -205,9 +206,9 @@ static symtable_header_t symt_open(void) {
  * @param idx       Index of the entry to return
  * @return addrtable_entry_t  Entry of the address table
  */
-static addrtable_entry_t symt_addrtab_entry(symtable_header_t *symt, int idx)
+static addrtable_entry_t symt_addrtab_entry(symtable_header_t *symt, unsigned int idx)
 {
-    assert(idx >= 0 && idx < symt->addrtab_size);
+    assert(idx < symt->addrtab_size);
```

Comment:
Is there a specific rationale for this int/unsigned changes? In general I prefer to use signed numbers because they have less suprising behaviors in case of bugs, I've had quite a few issues with unsigned having surprising behaviors.